bescort.lic

```diff
@@ -243,7 +249,17 @@ class Bescort
       pause 0.5
     end
   end
-
+	
+	def oshu_manor(mode) 
+		case mode
+		when /exit/i
+      walk_to(2317)
+    when /enter/i
+			wait_for_script_to_complete('oshu_manor')
+			find_room_list(%w(e e w w w n n n))
```

Comment:
I was surprised to see that the `find_room_list` logic does not match other logic for finding an empty room (e.g. it doesn't use the `hunting_buddies` setting). @rcuhljr might know if that is intentional or not...